Makefile

```diff
@@ -117,8 +121,8 @@ build-subpackages:
 	for subpackage in $(SUBPACKAGES); \
 	do \
 	    pushd $$subpackage; \
-	    $(PYTHON) ./setup.py clean --all \
-	    $(PYTHON) ./setup.py build --root=$(DESTDIR) --prefix=$(PREFIX); \
+	    $(PYTHON) ./setup.py clean --all; \
+	    $(PYTHON) ./setup.py build; \
```

Comment:
Why did you remove `--root` and `--prefix` arguments? Did it cause any issue?

Reply:
I think they were not listed as parameters in my case, but I might have to reevaluate that.

Reply:
To be fair, this was not the only thing I had to adapt for Ubuntu 20.04 (e.g. using `python3` instead of `python`), but this was the output I got:
```
for subpackage in syspurpose; \
do \
    pushd $subpackage; \
    python3 ./setup.py clean --all; \
    python3 ./setup.py build  --root=/ --prefix=/usr; \
    popd; \
done;
~/subscription-manager/syspurpose ~/subscription-manager
running clean
removing 'build/lib' (and everything under it)
'build/bdist.linux-x86_64' does not exist -- can't clean it
'build/scripts-3.8' does not exist -- can't clean it
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --root not recognized
~/subscription-manager
```

Reply:
OK, I see. Can you please make this change platform specific? We try to do some tests with virtual environments (#2228) and this change could break it.

Reply:
Came back to me now: `--root` and `--prefix` are valid parameters for the `install` subcommand.
So it should be save to remove them when using only the `build`-subcommand.

The `install`, which needs those options, is done in the `install-subpackages-via-setup`-stage:
https://github.com/candlepin/subscription-manager/blob/70d7c73d3a2bceef661c409ce687be3aa92c9b26/Makefile#L343-L349

Reply:
removed parameters only for debian-distributions.

I assume the `;` is save to append at the `setup.py clean` line!?